docker-compose.yml

```diff
@@ -60,6 +74,15 @@ services:
       - CELERY_RESULT_BACKEND=redis://redis:6379/0
       - CELERY_LOGLEVEL=DEBUG      
       - ES_HOST=elasticsearch
+      # Needed for Celery Flower to match the TIME_ZONE configured in the
+      # settings used by workers and beat containers.
+      - TZ=Europe/Amsterdam
+      # Legacy endpoints are deprecated, new installs should opt-out of using legacy
+      # settings (the default is to use legacy endpoints).
+      - USE_LEGACY_OIDC_ENDPOINTS=no
```

Comment:
These envvars are not relevant for OIP

Reply:
The OIDC endpoints? I thought these were part of the library?

Also the TZ can be gone? The comment seemed to indicate it was useful.

Reply:
the `USE_LEGACY_*` envvars are Open Forms specific, they're not part of the library. The breaking change will be done in OIP instead of adding an opt-in flag since nobody is using this yet, which is not the case for OF

TZ needs to stay for celery I believe, but my comment is only about the OIDC envvars!